src/config/mod.rs

```diff
@@ -287,10 +287,12 @@ impl Default for Alpha {
     }
 }
 
-#[derive(Debug, Copy, Clone, PartialEq, Eq)]
+#[derive(Debug, Copy, Clone, Deserialize, PartialEq, Eq)]
 pub enum Decorations {
     Full,
+    #[cfg(not(widows))]
     Transparent,
+    #[cfg(target_os = "macos")]
     Buttonless,
     None,
 }
```

Comment:
We should make these either case insensitive or change them to lowercase, otherwise we'll break existing configuration files.